File: Interfaces/ILibros.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Interfaces
{
    /// <summary>
    /// Estructura básica para el objeto Libros
    /// </summary>
    public interface ILibros
    {
        long Isbn { get; set; }
        string Titulo { get; set; }
        string Genero { get; set; }
        string NumeroPaginas { get; set; }
        int EditorialesId { get; set; }

    }
}

File: Spa/ClientApp/src/app/interfaces/ilibros.ts
import { IAutoresLibros } from "./iautores-libros";
import { IEditoriales } from "./ieditoriales";

export interface ILibros {
  isbn: number;
  titulo: string;
  genero: string;
  numeroPaginas: string;
  editorialesId: number;
  editorial: IEditoriales;
  autoresLibros: IAutoresLibros;
}
